libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -331,6 +355,11 @@ void AP_OSD_Screen::draw_horizon(uint8_t x, uint8_t y)
     float roll = ahrs.roll;
     float pitch = -ahrs.pitch;
 
+    //inverted roll AH
+    if (osd->options.get() & AP_OSD::OPTION_INVERTED_AH_ROLL) {
```

Comment:
this would become if (get_options() & AP_OSD::OPTION_INVERTED_AH_ROLL)

Reply:
or maybe a check_option() function that returns a bool?

Reply:
Thanks, good idea. Implemented.